data/human/human missions.txt

```diff
@@ -5416,3 +5416,194 @@ mission "Deal Change 2"
+			`			Your father."`
+			`	Understanding the personal nature of the message, you leave the two of them alone for a bit, walking around and looking at the artifacts on the left of the dome.`
+			`	After some time, the siblings call you back to them. Sophie says, "Captain <last>, Liam and I have thought through about this, and we think that we would never be able to live up to our father's expectations if if we kept this sword. We can study all these artifacts and understand more about the universe through them, but we cannot study this one. To understand it, it needs to be used.`
+			`	"This sword was always at our father's side, and it has been many places. Take it to many more." She pulls the sword from the pedestal, and gives it one last look over before handing over it to you. "It is yours now."`
+			choice
+				`	"I would be more than grateful to accept it."`
+						goto accepted
+				`	"This is too much for me to take."`
+						goto lastchance
+			label lastchance
+			`	"Are you sure?" Sophie asks. "This is a one time offer. It would be wasted on us."`
+			choice
+				`	"Yes, I'm sure."`
+					goto declined
+				`	"Well, if you frame it that way, I will take it."`
+			label accepted
+			action
+				outfit "Old Man's Sword" 1
+				payment 50000
+			`	"Well, we'll need some time to take it all in and arrange things, find a place to stay on Prime and all that," Sophie says, "but we won't keep you from your ship any longer. It was a pleasure to travel with you, Captain <last>."`
+			`	Liam adds, "We will always thank you for helping us with our father's will, we never imagined that we would come so far and we would have never done it without you. It means so much to us. We wish you well through the stars, Captain <last>."`
+			`	After Liam hands you a credit chip of <payment>, you say goodbye to the two siblings and leave them in their father's mysterious dome.`
+				decline
+			label declined
+			action
+				payment 100000
+			`	"I see... please, if you won't take our father's sword, take an extra payment for having to put up with us all the way up until now." She takes the sword away and inserts it back into the pedestal.`
+			`	"Well, we'll need some time to take it all in and arrange things, find a place to stay on Prime and all that," Sophie says, "but we won't keep you from your ship any longer. It was a pleasure to travel with you, Captain <last>."`
+			`	Liam adds, "We will always thank you for helping us with our father's will, we never imagined that we would come so far and we would have never done it without you. It means so much to us. We wish you well through the stars, Captain <last>."`
+			`	After Liam hands you a few credit chips for <payment>, you say goodbye to the two siblings and leave them in their father's mysterious dome.`
```

Comment:
This is identical to the paragraph above. You can just delete this and replace it with a goto message that will direct to the paragraph above.

Reply:
```suggestion
			`	After Liam hands you a few credit chips for 100,000 credits, you say goodbye to the two siblings and leave them in their father's mysterious dome.`
```
On top of the <payment> having a difference between "a credit chip"/"a few credit chips" making them different, we'll also probably need to hard-write the payments since the substitution can't detect them being updated on the fly.

Reply:
I remember when I was testing if this worked that it did update, but sure, hard-writing it is fine.